ports/rp2/pendsv.c

```diff
@@ -76,7 +114,7 @@ void pendsv_resume(void) {
 
 void pendsv_schedule_dispatch(size_t slot, pendsv_dispatch_t f) {
     pendsv_dispatch_table[slot] = f;
-    if (pendsv_mutex.mutex.enter_count == 0) {
+    if (pendsv_suspend_count() == 0) {
```

Comment:
Maybe call it `pendsv_lock_count`?

Reply:
This function returns the count of times `pendsv_suspend()` has been called, but I can change the name if you prefer.

Reply:
Oh, right.  Keep it as `pendsv_suspend_count()`.